bootstrapper/src/contract_clients/eth_bridge.rs

```
@@ -223,20 +222,9 @@ impl StarknetLegacyEthBridge {
         rpc_provider: &JsonRpcClient<HttpTransport>,
         l2_bridge_address: Felt,
         erc20_address: Felt,
-        l2_deployer_address: &str,
+        _l2_deployer_address: &str,
         account: &RpcAccount<'_>,
     ) {
-        let tx = invoke_contract(
-            l2_bridge_address,
-            "initialize",
-            vec![Felt::from_dec_str("1").unwrap(), Felt::from_hex(l2_deployer_address).unwrap()],
-            account,
-        )
-        .await;
-
-        log::info!("🎡 setup_l2_bridge : l2 bridge initialized //");
-        wait_for_transaction(rpc_provider, tx.transaction_hash, "setup_l2_bridge : initialize").await.unwrap();
```

We don't deploy the bridge anymore?

---

This was not needed before too, but this was failing silently sadly. Now we have have properly fetch the receipt hence this was catched

---

We do deploy the bridge, the initialise is automatically called when upgrading the bridge